spec/rubocop/cop/rspec/indexed_let_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::IndexedLet do
+  let(:max_repeats) { 1 }
+  let(:cop_config) { { 'MaxRepeats' => max_repeats } }
+
+  specify do
```

Comment:
It sometimes outputs like this, so it seems better to pass a description to it or specify like other specs.
```
RuboCop::Cop::RSpec::IndexedLet
  is expected to eq describe SomeService do
  let(:item_1) { create(:item) }
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ This `let` ...ame. Please give it a meaningful names, use create_list or move creation to the `before` block.
end
```